docs/SQL/gems/transformations/deduplicate.md

```diff
@@ -0,0 +1,83 @@
+| Deduplicate On Columns | Columns to consider while removing duplicate rows (not required for `Distinct Rows`)                                                                                                                                                                                                                                       | True     |
+
+## Row to keep options
+
+As mentioned in the previous parameters, there are four Row to keep options that you can use in your deduplicate Gem.
+
+![Deduplicate row to keep](./img/deduplicate_row_to_keep.png)
+
+In the Code view, you can see that the Deduplicate Gem contains `SELECT DISTINCT *` when using the `Distinct Rows` option.
+
+![Deduplicate code view](./img/deduplicate_code_view.png)
+
+## Example
+
+Suppose you're deduplicating the following table.
+
+| First_Name | Last_Name | Type  | Contact           |
+| :--------- | :-------- | :---- | :---------------- |
+| John       | Doe       | phone | 123-456-7890      |
+| John       | Doe       | phone | 123-456-7890      |
+| John       | Doe       | phone | 123-456-7890      |
+| Alice      | Johnson   | phone | 246-135-0987      |
+| Alice      | Johnson   | phone | 246-135-0987      |
+| Alice      | Johnson   | email | alice@johnson.com |
+| Alice      | Johnson   | email | alice@johnson.com |
+| Bob        | Smith     | email | bob@smith.com     |
+
+For `Distinct Rows`, the interim data will show the following:
+
+| First_Name | Last_Name | Type  | Contact           |
+| :--------- | :-------- | :---- | :---------------- |
+| John       | Doe       | phone | 123-456-7890      |
+| Alice      | Johnson   | phone | 246-135-0987      |
+| Alice      | Johnson   | email | alice@johnson.com |
+| Bob        | Smith     | email | bob@smith.com     |
+
+The `First` and `Last` options work similarly to `Distinct Rows`, but they keep the first and last occurrence of the duplicate rows respectively.
+
+For `Unique Only`, the interim data will look like the following:
+
+| First_Name | Last_Name | Type  | Contact       |
+| :--------- | :-------- | :---- | :------------ |
+| Bob        | Smith     | email | bob@smith.com |
+
+You'll be left with only one unique row since the rest were all duplicates.
+
+---
+
+You can add `First_Name` and `Last_Name` to Deduplicate On Columns if you want to further deduplicate the table.
+
+For `Distinct Rows`, the interim data will show the following:
+
+| First_Name | Last_Name |
+| :--------- | :-------- |
+| John       | Doe       |
+| Alice      | Johnson   |
+| Bob        | Smith     |
+
+For `Unique Only`, the interim data will look like the following:
+
+| First_Name | Last_Name |
+| :--------- | :-------- |
+| Bob        | Smith     |
```

Comment:
for unique_only/first/last case, the interim data will contain all columns irrespective what columns were selected

Reply:
So it would look the same as the table on line 50?

Reply:
same as line 63

Reply:
```
| First_Name | Last_Name | Type  | Contact       |
| :--------- | :-------- | :---- | :------------ |
| Bob        | Smith     | email | bob@smith.com |
```

Reply:
@alexanderahn yes, based on your edited comment 😅

Reply:
Done.